Makefile

```diff
@@ -121,11 +130,14 @@ install: crd
 .PHONY: deploy
 deploy: manifests install generate
 	# Deploy the operator manifests:
+	oc create namespace ${HIVE_OPERATOR_NS} || true
 	mkdir -p overlays/deploy
 	cp overlays/template/kustomization.yaml overlays/deploy
-	cd overlays/deploy && kustomize edit set image registry.svc.ci.openshift.org/openshift/hive-v4.0:hive=${DEPLOY_IMAGE}
+	cd overlays/deploy && kustomize edit set image registry.svc.ci.openshift.org/openshift/hive-v4.0:hive=${DEPLOY_IMAGE} && kustomize edit set namespace ${HIVE_OPERATOR_NS}
 	kustomize build overlays/deploy | oc apply -f -
 	rm -rf overlays/deploy
+	# Create a default basic HiveConfig so the operator will deploy Hive
+	oc process --local=true -p HIVE_NS=${HIVE_NS} -p LOG_LEVEL=${LOG_LEVEL} -f config/templates/hiveconfig.yaml | oc apply -f -
```

Comment:
This still makes the debug the default log level. I can live with this for now. I don't know who else uses make deploy outside of OpenShift development.